### ISSUE TITLE
Bridge compiler should detect when project references non-Bridge projects and give a suitable error

### ISSUE THREAD

**Issue by DanTup** [bridgedotnet/Bridge#2555]
A few times my colleagues have added references to our other C# projects in order to reuse code (this is a mistake, and they should be using linked files). When they do, the compilation seems to succeed (though there are other issues).

The project I just opened had references to several `System.*` DLLs as well as `JSON.NET` NuGet package and others!

Is it possible for Bridge to detect when this happens and give an obvious compile error?

**Comment by geoffreymcgill** [comment]
I agree the current situation is far from ideal. This is something we need to work on. 

I have some ideas, although an implementation is going to take some time.

**Comment by michaelcheers** [comment]
Duplicate of #46.
